aks/variables.tf

```diff
@@ -10,7 +10,7 @@ variable "node_count" {
     default = "1"
 }
 variable "machine_type" {
-    default = "Standard_DS4_v2"
+    default = "Standard_DS3_v2"
```

Comment:
Our recommendation is to use DS4_v2 though...on DS3 we have seen bunch of test failures on HA